lesson1.md

```diff
@@ -350,143 +312,110 @@ because the operators in JavaScript behave differently depending on the data typ
 
 ---
 
-### Let's practice
 
-Compute the following with JavaScript:
+### Data Type: Boolean
 
-* How old you will be in one year?
-* What is the difference between your age and the person next to you?
-* What is the square of 8?
+* **Boolean** describes a yes/no situation
+* in Javascript, we use `true` or `false`
 
 ---
 
-### Solution
 
-```javascript
-42 + 1     // I'll be 43 next year!
-42 - 20    // I'm 22 years older
-8 ** 2     // Square of eight is 64
-```
+### DIY?
+
+Open the console, and type in the following:
+
+* true
+* false
+
+Yep, that is it.
+
+It seems not very useful...
+
+YOU JUST WAIT!
+<!-- .element: class="fragment" -->
+
 
 ---
 
 ### Comparison operators
 
+For all data types:
 * `===` strict equality
 * `!==` strict inequality
+
+Numbers only:
 * `>` greater than
 * `<` less than
 * `>=` greater or equal
 * `<=` less or equal
 
+All of these return a boolean!
+
 ---
 
 ### Let's practice
 
-Use JavaScript to compute the following:
-
-* Figure out whether you are older than the person next to you
-* Figure out whether you have the same first name as the person next to you
+* Are you older than the person next to your right?
+* Do you have the same first name as the person next to you?
 
 ---
 
 ### Solution
 
-```javascript
-42 > 20                   // true - I am older
+```js
+42 > 40                   // true - I am older
 "Harald" === "Abdullah"   // false - not the same first name
 ```
 
 ---
 
+
 ### Logical Operators
 
-These operators only make sense on Boolean:
+These operators also work only on Boolean values:
 
+* `!` Logical NOT
 * `&&` Logical AND
 * `||` Logical OR
-* `!` Logical NOT
-
----
-
-### Logical AND
-
-AND takes two booleans and returns `true` if both booleans are `true`
-
-```javascript
-true && true   // true
-true && false  // false
-false && true  // false
-false && false // false
-```
-
-Example: I need bread (true/false) AND cheese (true/false) to make a cheese toast
-
----
-
-### Logical OR
-
-OR takes two booleans and returns `true` if any (or both) are `true`
-
-```javascript
-true || true   // true
-true || false  // true
-false || true  // true
-false || false // false
-```
 
-Example: To apply to a job, I need to know German OR English.
+We are not going to explain them today, but definitely when we talk about conditionals.
```

Comment:
Any reason for not talking about them in this lesson. Since in the summary part, there is even an example of logical operators.

Reply:
I don't think we have enough time for them, also, in comparison with everything else, they are confusing.